main.tex

```diff
@@ -18,6 +18,21 @@
 \begin{document}
 
 \maketitle
+\subsubsection Internetverbreitung
+
+In Indien gibt es 2015 schon 350 millionen Menschen\autocite{slideshareIndia} , welche das Internet aktiv nutzen, und davon sind 159 millionen per Handy im Internet unterwegs. Somit sind sie schon auf Platz 2 der aktiven Internetnutzer weltweit. So groß sich die Zahl auch anhören mag, das sind lediglich 27\% der Bevölkerung\autocite{InternetCountry}. 
+
+Laut einer Studie sei 2015 ein PC in Indien ein Luxusgut, welches durch die schnelle Entwicklung des Handys für die Jugend schnell die Bedeutung verlor, und Handys mit Internet in der Gesellschaft allgegenwärtig wurde. Durch die explosionsartige Verbreitung der neuen Möglichkeit mit anderen zu kommunizieren und sich auszutauschen bauten sich in kurzer Zeit große Gruppen im Internet auf, um sich über neue Technologien, Wissenswertes und Interessen auszutauschen. Dieser rapide Aufbau von Gesellschaften ist ein neues Phänomen, welches vorallem in "Entwicklungsländern" auftritt, in denen neue und vergleichsweise preiswerte Technologien den Zugang zum Internet ermöglichen.   \textcite{empowermentThroughFacebook}
```

Comment:
```suggestion
Ein Computer ist in Indien ein Luxusgut, welches durch die rasante Entwicklung von Mobilgeräten für die Jugend schnell an Bedeutung verlor, während Handys mit mobilem Internet allgegenwärtig wurden.
Durch die explosionsartige Verbreitung der neuen Kommunikationsmöglichkeiten entstanden in kürester Zeit große Gruppen im Internet, welche sich über neue Technologien, Wissenswertes und Ihre Interessen auszutauschen.
Dieser rapide Aufbau von Gesellschaften ist ein neues Phänomen, welches vorallem in `Entwicklungsländern' auftritt, in denen neue, und vergleichsweise preiswerte Technologien, den Zugang zum Internet ermöglichen.   \textcite{empowermentThroughFacebook}
```

Reply:
`rapide Aufbau von Gesellschaften` -> `rapide Aufbau von gesellschaftlichen Strukturen`?

Reply:
Wenn das Quelle Facebook, also Studie bezahlt von Facebook, das vlt. in den Text mit einfließen lassen.
Z.b.: `Laut einer von Facebook in Auftrag gegebenen Studie:`

Reply:
Ich glaube die Studie, welche ich hier benutzt habe, war nicht von Facebook in Auftrag gestellt worden, kann aber gerne den Satz ggf. nochmal umformulieren :)